pkgs/development/tools/castxml/default.nix

```diff
@@ -64,5 +64,6 @@ stdenv.mkDerivation rec {
     license = licenses.asl20;
     maintainers = with maintainers; [ AndersonTorres ];
     platforms = platforms.unix;
+    broken = stdenv.isDarwin;
```

Comment:
This is fixed on staging-next

Reply:
Will it be superceded, then?

Reply:
I've pushed a revert a3f99775dab6584271881171003ab55a06381a80. Still, thanks for watching the darwin build status.